### PR TITLE
Pull Request to allow Bootstrap 4's testing in CakePHP sandbox

### PR DESCRIPTION
**Pull request by Modicrumb**
I have changed items to be able to include bootstrap 4 within your sandbox.

### CODE REVIEW

File: plugins/Sandbox/src/Controller/BootstrapController.php [hunk 1]
```diff
@@ -27,6 +27,10 @@ public function beforeFilter(Event $event) {
 
 		$this->Auth->allow();
 		parent::beforeFilter($event);
+
+		if ($this->request->pass[0] == 'v4') {
```
Comment (dereuromark) [inline plugins/Sandbox/src/Controller/BootstrapController.php]: shouldnt that check on the existence first (!empty())?
PS: Please `===` always

File: plugins/Sandbox/src/Template/Bootstrap/index.ctp [hunk 2]
```diff
@@ -1,3 +1,7 @@
 <h2>Bootstrap V3</h2>
 
 <?php echo $this->element('Sandbox.actions'); ?>
+
+<h2>Bootstrap V4</h2>
+
+<?php echo $this->element('Sandbox.actions',['arguments'=>['v4'] ]); ?>
```
Comment (dereuromark) [inline plugins/Sandbox/src/Template/Bootstrap/index.ctp]: space after comma missing and no space after ]

File: plugins/Sandbox/src/Template/Element/actions.ctp [hunk 3]
```diff
@@ -1,5 +1,10 @@
+<?php
+ 	if (!isset($arguments)) {
+		$arguments = [];
+	}
+?>
 <ul>
 <?php foreach ($actions as $action) { ?>
-	<li><?php echo $this->Html->link(\Cake\Utility\Inflector::humanize($action), ['action' => $action]); ?></li>
+	<li><?php echo $this->Html->link(\Cake\Utility\Inflector::humanize($action), array_merge(['action' => $action],$arguments) ); ?></li>
```
Comment (dereuromark) [inline plugins/Sandbox/src/Template/Element/actions.ctp]: space after comma

**Comment by dereuromark** [comment]
I wonder if there is a cleaner way than just copying the whole layout file.
Like an internal switch there.

**Comment by Modicrumb** [comment]
I've made the changes requested as well as added a switch statement in default layout to handle different potential assets.

**Comment by dereuromark** [comment]
I updated the server, but it doesnt look too good yet: http://sandbox3.dereuromark.de/sandbox/bootstrap

**Pull request merged**